.gitignore

```
@@ -169,3 +169,6 @@ cython_debug/
 #  and can be added to the global gitignore or merged into this file.  For a more nuclear
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
 #.idea/
+
+# Auto generated by setuptools scm
+virtual_ship/_version_setup.py
```

Add a hard return at tend of file
```suggestion
virtual_ship/_version_setup.py

```